test/testmetrics/metrics-hpa/install.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: podtato-head-entry
+  namespace: $NAMESPACE
+  labels:
+    app: podtato-head
+spec:
+  selector:
+    matchLabels:
+      component: podtato-head-entry
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        component: podtato-head-entry
+    spec:
+      terminationGracePeriodSeconds: 5
+      containers:
+        - name: server
+          image: ghcr.io/podtato-head/entry:0.2.8
+          imagePullPolicy: Always
+          resources:
+            limits:
+              cpu: 5m
```

Comment:
the requests/limits for the cpu can be increased a bit since we are using data from the mock server. they were set very low in the example to provoke a high cpu saturation value, but that resulted in the pod taking quite a long time to start up, so increasing the cpu resource might help the test to finish a bit faster